images/releng/ci/variants.yaml

```diff
@@ -6,11 +6,6 @@ variants:
     REVISION: '0'
   go1.20-bookworm:
     CONFIG: 'go1.20-bookworm'
-    GO_VERSION: '1.20.5'
-    OS_CODENAME: 'bookworm'
-    REVISION: '0'
-  go1.19-bookworm:
```

Comment:
I think the releng one we can keep until 1.21 is out but not strong opinion on this